repo/format/format_manager_test.go

```diff
@@ -53,15 +55,15 @@ func TestFormatManager(t *testing.T) {
 	startTime := time.Date(2020, 1, 1, 12, 0, 0, 0, time.UTC)
 	ta := faketime.NewTimeAdvance(startTime, 0)
 	nowFunc := ta.NowFunc()
-	cache := format.NewMemoryBlobCache(nowFunc)
+	blobCache := format.NewMemoryBlobCache(nowFunc)
```

Comment:
@ashmrtn Can you split this PR in 2 please?
Have the blobCache renaming refactoring in one, and the functional change with the tests in another one? Thanks.

Reply:
unfortunately I can't :/ the renaming is only to get around linting errors and should be in the final commit in the series. Without it the linter complains there's a variable named the same as an import (the cache package since that's what I use in the newer tests

Reply:
Ashlie, I took a quick stab and split it two:
1. #3211
2. julio-lopez/kopia/pull/332

I'll merge #3211 soon. After that you can pull the commits from the second PR and either push them to this PR (preferred) or open a new PR. Your choice.

Thanks.

Reply:
Well, I actually missed a few renames in the first PR :(

Reply:
ah, sorry, I misunderstood the ordering request in your first comment (I guess that's what I get for reading comments before my morning caffeine). Thanks for the assist on that one. I'll pull the new commits in to this one

Reply:
No worries. I had not realized that the renaming was prompted by the linter and then necessary to get the PR passing. 😃